### DISCUSSION TITLE
How to noobify a great resource.

### DISCUSSION THREAD

**Discussion by joehodgy** [collin80/SavvyCAN#283] [Q&A] [answered]
Hi Colin, I'm new to github but not new to CAN or Savvycan itself - how/where do I dl and install the *latest* level software for SavvyCan? I have the 199 win 32 version currently. Thanks

**Comment by FrankM54** [comment, marked as answer]
I am also new to Github and had trouble finding it as well.

At the "collin80/SavvyCAN" page on the right side of the screen just below "About" is "Releases" then select the version you want.
or just click here:
https://github.com/collin80/SavvyCAN/releases

**Reply by collin80** [reply]
Thanks. Yes, the releases tab has all of the versions from now on. The "continuous" build is done every time any commit is made to GitHub. That means it is the absolute most up to date version. This may or may not be a good thing. But, generally I try not to have the state of the program broken when commits are done here. So, usually the continuous build is the way to go.

**Reply by joehodgy** [reply]
Thanks both - sorry for the basic question!

Joe

**Reply by collin80** [reply]
No need to apologize at all! It isn't totally clear how you could find the current builds. I did just recently add a blurb about that to savvycan.com but otherwise it wasn't really spelled out. Besides, asking questions that get you closer to your goals is never a bad idea.
